CHANGES.md

```diff
@@ -74,8 +82,11 @@ To be released.
  -  Added `BlockChain<T>.GetBalance()` method.  [[#861], [#900]]
  -  Added `Block<T>.TotalDifficulty` property.  [[#666], [#917]]
  -  Added `SwarmOptions` class.  [[#926]]
- -  Added `PeerChainState` struct.  [[#936]]
- -  Added `Swarm<T>.GetPeerChainStateAsync()` method.  [[#936]]
```

Comment:
Apparently it went something wrong during you rebased.

Reply:
restore at https://github.com/planetarium/libplanet/pull/935/commits/29a125346fe298c2ad1fd634271cecece7a1d0fd